shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/scaling/core/job/importer/DefaultImporterCreator.java

```diff
@@ -27,22 +27,27 @@
 import org.apache.shardingsphere.data.pipeline.spi.importer.Importer;
 
 /**
- * Importer factory.
+ * Default importer creator.
  */
-@NoArgsConstructor(access = AccessLevel.PRIVATE)
-public final class ImporterFactory {
+public class DefaultImporterCreator implements ImporterCreator {
     
-    /**
-     * Create importer.
-     *
-     * @param importerConfig importer configuration
-     * @param dataSourceManager data source manager
-     * @param channel channel
-     * @param jobProgressListener job progress listener
-     * @return importer
-     */
-    public static Importer createImporter(final ImporterConfiguration importerConfig, final PipelineDataSourceManager dataSourceManager, final PipelineChannel channel,
-                                          final PipelineJobProgressListener jobProgressListener) {
+    @Override
+    public String getType() {
+        return "MySQL";
+    }
+    
+    @Override
+    public Collection<String> getTypeAliases() {
+        Collection<String> aliases = new LinkedList<>();
+        aliases.add("PostgreSQL");
+        aliases.add("openGauss");
+        return aliases;
+    }
```

Comment:
It's better to put getType() and getTypeAliases() at the bottom of class, conventionally